src/Properties/SchemaColumn.php

```diff
@@ -3,12 +3,12 @@
 declare(strict_types=1);
 
 /**
- * This file is part of Larastan.
+ * This file uses parts of code from following projects.
```

Comment:
I would remove this too.

Reply:
Maybe just add a note on the class phpdocs.

Reply:
Updated.